tests/ludwig/schema/test_utils.py

```diff
@@ -0,0 +1,8 @@
+from ludwig.constants import TYPE
+from ludwig.schema import utils as schema_utils
+
+
+def test_remove_duplicate_fields():
+    props = {TYPE: "random"}
+    schema_utils.remove_duplicate_fields(props, [TYPE])
+    assert TYPE not in props
```

Comment:
We should test the positive case as well: a property not listed in the array continues to remain in the `props` object.